### DISCUSSION TITLE
[BUG] Install e-commerce

### DISCUSSION THREAD

**Discussion by badaroerik** [algolia/instantsearch#5612] [General]
### To install them, you can run: 
``npm install --save instantsearch.js/es/Connectors instantsearch.js/es/lib/routers Vue-instantsearch
C% ``

### When I try to install it appears:

npm ERR! code ENOENT
npm ERR! open syscall
npm ERR! Errno -2
npm ERR! enoent ENOENT: no file or directory, open....
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent  

Does anyone know how to solve it?

**Comment by dhayab** [comment]
Hi, because the example is part of the monorepo, it relies on the other packages being built in order to work correctly.

To do that, type `yarn build` or `npm run build` before serving the e-commerce project.

If you want to rely on packages already released, you can also copy the example outside of the monorepo.
